client/src/components/NavBar.tsx

```diff
@@ -8,26 +8,23 @@ export interface NavbarProps {
   userName: string
   userAvatarHash: string
 }
-
-export class NavBar extends React.Component<NavbarProps> {
-  // TODO the real avatar image is "https://cdn.discordapp.com/avatars/"userId"/"userAvatarHash".png";
-  render(): ReactNode {
-    return (
-      <Navbar bg="dark" variant="dark">
-        <Navbar.Brand href="tournaments">Discord League</Navbar.Brand>
-        <Nav.Link as={Link} to="tournaments">
-          Tournaments
-        </Nav.Link>
-        <Nav.Link as={Link} to="my-games">
-          My Games
-        </Nav.Link>
-        <Navbar.Collapse className="justify-content-end">
-          <Navbar.Text>
-            Signed in as: <Link to="/login">{this.props.userName}</Link>
-          </Navbar.Text>
-          <Avatar src="https://i.pinimg.com/236x/42/0f/32/420f32cbff710bb30486a88353fa7ee1.jpg" />
-        </Navbar.Collapse>
-      </Navbar>
-    )
-  }
+// TODO the real avatar image is "https://cdn.discordapp.com/avatars/"userId"/"userAvatarHash".png";
+export function NavBar(props: NavbarProps): JSX.Element {
```

Comment:
I prefer the style of writting like:

```ts
 export function NavBar({ userName, userAvatarHash }: { userName: string; userAvatarHash: string })
```

I find it easier to keep track of stuff, cause then the props are closer to where they get used